crates/sui-core/src/authority/authority_store.rs

```diff
@@ -738,6 +738,32 @@ impl AuthorityStore {
         Ok(())
     }
 
+    /// Insert objects directly into the object table, but do not touch other tables.
+    /// This is used in fullnode to insert objects from validators certificate handling response
+    /// in fast path execution.
+    pub(crate) fn insert_objects_to_object_store(&self, objects: &Vec<Object>) -> SuiResult {
```

Comment:
there is no guarantee that these objects must not be owned objects, even though they are only clock objects today.
So I believe instead of duplicating the code and defining a new function we should see if we could just reuse the bulk insert method

Reply:
I have to make some modification to the function as @phoenix-o suggested above, which will make the function different from `bulk_insert` at first glance. But agree we should clean these things up soon.